### DISCUSSION TITLE
Wrap lines in Code Block and adjust line numbers to reflect wrapping?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [facelessuser/pymdown-extensions#1594] [Q&A] [answered]
As first discussed in https://github.com/squidfunk/mkdocs-material/discussions/3541, is there a way to mimic this code formatting style where if a line wraps, the line number is not incremented? See line 64.

<img width="1106" alt="Screen Shot 2022-02-05 at 6 46 36 am" src="https://user-images.githubusercontent.com/9379524/152601356-a8f326e3-043b-4df5-887a-91fae551111f.png">
 
I've used CSS to get close but am not sure how to adjust line numbering 

```
code {
  white-space : pre-wrap !important;
  word-break: break-word;
}
```

Line 106 is the equivalent to  line 64 above, but the line number has been incremented, rather than shown as a dot •

<img width="1183" alt="Screen Shot 2022-02-05 at 6 56 46 am" src="https://user-images.githubusercontent.com/9379524/152602122-b6107be9-4fc3-4a3f-af9d-51eea121618a.png">

Thanks in advance.

**Comment by facelessuser** [comment, marked as answer]
I actually just posted a response over at https://github.com/squidfunk/mkdocs-material/discussions/3541. You may find some help in trying to achieve what you are looking for over here: https://github.com/facelessuser/pymdown-extensions/issues/1138. Someone was attempting such changes. I only enabled the features, I never attempted to achieve their CSS solution, and any CSS found in that discussion may need tweaking to work with Material.

**Reply by Stephen-Gates** [reply]
Sorry for double posting and thanks for your quick reply 🙏
